migrationrunners/migrationrunners.go

```diff
@@ -26,25 +27,16 @@ func New(server servers.IServer) (*Runner, error) {
 
 // RunOutstanding runs all outstanding migrations
 func (r *Runner) RunOutstanding() error {
-	migrationRepo, err := migrationloaders.Load()
+	migrationRepo, err := r.getOutstandingMigrations()
```

Comment:
This got factored out to support a check command that also needs to see outstanding migrations to simulate their application to the schema, but it's tidy so I'm keeping it.